ansible_runner/runner_config.py

```diff
@@ -407,6 +429,27 @@ def generate_ansible_command(self):
 
         exec_list = [base_command]
 
+        if self.cli_execenv_cmd:
```

Comment:
Why would they fail? We haven't needed these before.

Reply:
Even accepting the need for the hack, the way this is implemented concerns me, because these will be defined in the extravars file, but `-e` with dummy data could still get added in conflict with those.